Wikipedia/Code/WMFTableOfContentsItem.swift

```diff
@@ -40,6 +40,10 @@ public enum TableOfContentsItemType {
 public enum TableOfContentsBorderType {
     case Default
     case FullWidth
+    case DefaultTopOnly
+    case FullWidthTopOnly
+    case DefaultBottomOnly
+    case FullWidthBottomOnly
     case None
 }
```

Comment:
@joewalsh 
Looks like we only use None, Default and DefaultTopOnly. Thoughts on maybe removing the unused cases?